browser-test/src/trusted_intermediary.test.ts

```diff
@@ -215,7 +219,7 @@ test.describe('Trusted intermediaries', () => {
     await page.waitForSelector('h2:has-text("Edit Client")')
     await page.click('text=Back to client list')
     await waitForPageJsLoad(page)
-    await page.waitForSelector('h2:has-text("Add Client")')
+    await page.waitForSelector('h4:has-text("Search")')
     await validateScreenshot(page, 'back-link-leads-to-ti-dashboard')
```

Comment:
suggestion (non-blocking):  The screenshot seems unnecessary here since we aren't testing any of the styles on the dashboard page.  The selector on line 222 should be sufficient to know that the back link is working.